src/components/remote_control/include/remote_control/resource_allocation_manager_impl.h

```diff
@@ -18,7 +18,8 @@ class ResourceAllocationManagerImpl : public ResourceAllocationManager {
                  const uint32_t hmi_app_id,
                  AskDriverCallBackPtr callback) OVERRIDE FINAL;
 
-  void SetAccessMode(const AccessMode::eType access_mode);
+  void SetAccessMode(
+      const hmi_apis::Common_RCAccessMode::eType access_mode) OVERRIDE FINAL;
```

Comment:
@AByzhynar please see the latest updates ed6379f1300d0f2493206a1ed7e0f581e8cc764a